### PR TITLE
Fix docker build, other fixes

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -81,7 +81,7 @@ if(ROCKETS_FOUND AND ROCKETS_USE_LIBWEBSOCKETS)
     list(APPEND COMMON_FIND_PACKAGE_DEFINES BRAYNS_USE_NETWORKING)
   endif()
 else()
-  unset(BRAYNS_NETWORKING_ENABLED)
+  set(BRAYNS_NETWORKING_ENABLED OFF)
```
Comment (hernando) [inline CMakeLists.txt]: `set(BRAYNS_NETWORKING_ENABLED)` should also work, doesn't it?

Comment (tribal-tec) [reply CMakeLists.txt]: Don't know, maybe? The explicit one is more obvious

Comment (tribal-tec) [reply CMakeLists.txt]: According to documentation it has the semantics as `unset`, hence the variable is uninitialized. But as it was set to `ON` before, this has no effect in practice, only an explicit set to `OFF`.

**Comment by tribal-tec** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by tribal-tec** [comment]
retest this please

**Comment by tribal-tec** [comment]
ping

**Pull request merged**